tests/test_bonfire.py

```diff
@@ -254,6 +254,7 @@ def test_ns_reserve_flag_timeout(mocker, caplog, user: str, namespace: str, time
     mocker.patch("bonfire.bonfire.get_pool_size_limit", return_value=0)
     mocker.patch("bonfire.bonfire.has_ns_operator", return_value=True)
     mocker.patch("bonfire.namespaces.whoami", return_value=user)
+    mocker.patch("bonfire.bonfire.get_namespace_pools", return_value=conf.DEFAULT_NAMESPACE_POOL)
```

Comment:
You need to mock this to return a list of strings. E.x. `return_value=["my", "fake", "pools"]`